lib/reporter.ts

```diff
@@ -198,6 +198,14 @@ class ReportPortalReporter extends Reporter {
         level: LEVEL.ERROR,
         message,
       });
+
+      if (this.options.cucumberNestedSteps) {
+        const suiteItem = this.storage.getCurrentSuite();
```

Comment:
Would not you mind call it not suite but testItem\scenarioItem? Cuz it is not `suite` as I get and it is not so obviously why we add log item to suite here..
Or I get smth wrong?

Reply:
Originally I named it `scenarioItem`, but changed it back to `suiteItem` since the function call is `getCurrentSuite`.  I'm good with calling it `scenarioItem`, I don't have much of a preference.

This is definitely confusing case that appears to come up with addition of nested steps in  ReportPortal v5.  If we only log at the test level (for cucumber this is the scenario steps) everything shows correctly in RP where the error message shows for the step and at the scenario (suite) level.  However, it appears that the auto-analyzer is not able to read the error log from the nested steps.  The code here adds the error log to the scenario (suite) level in RP.  Once added, the auto-analyzer has access to the error log and functions as expected.  

This might not be the best place for the fix, maybe we should look at creating an issue in the RP repo that the logs for nested steps aren't being taken into account for the auto-analyzer?  Then we could either reject this or use it as a short term work around until it's fixed.  What are your thoughts?

Reply:
>However, it appears that the auto-analyzer is not able to read the error log from the nested steps. 
>This might not be the best place for the fix, maybe we should look at creating an issue in the RP repo that the logs for nested steps aren't being taken into account for the auto-analyzer? 

Agree, it is strange. It seems a good question to report portal team, will ask about this.

>Then we could either reject this or use it as a short term work around until it's fixed.

yes it is ok as work around

Reply:
I've received answer. It will be fixed. Target version with fix is Report portal v.5.1.

So we can use this workaround until version with fix.